font/ttf.ksy

```diff
@@ -750,7 +915,29 @@ types:
         doc: 'The descender metric for Windows.'
       - id: code_page_range
         type: code_page_range
+        if: version > 0
         doc: 'This field is used to specify the code pages encompassed by the font file in the `cmap` subtable for platform 3, encoding ID 1 (Microsoft platform).'
+      - id: x_height
+        type: u2
+        if: version > 1
```

Comment:
Wouldn't it make more sense to use the "greater than _or equal to_" (e.g. <code>if: version ><ins><b>=</b></ins> 5</code>) operator rather than the non-inclusive "greater than" (<code>if: version > 4</code>)?

I think that format docs usually introduce some feature _since_ some format version (**including** the version when it was introduced), which suggests the `>=` operator.

Reply:
This makes sense, I will change it.

Reply:
Especially `if: version > 0` looks weird to me - that's equivalent to <code>if: version <ins><b>!=</b></ins> 0</code> (because `version` is unsigned), which suggests that the documentation literally says that the `version == 0` is a special case where some structure is **not present**, otherwise it is (in any other version).